### PR TITLE
[SD-4444][SD-4452] Show more than 25 products in the edit subscriber config, sort product listing

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

**Comment by mdhaman** [comment]
:+1:

File: scripts/superdesk-publish/publish.js [hunk 1]
```diff
@@ -433,10 +458,10 @@
                  * @return {*}
                  */
                 var fetchProducts = function() {
-                    return api.query('products').then(function(products) {
-                        $scope.availableProducts = products._items;
+                    return adminPublishSettingsService.fetchAllProducts().then(function(products) {
```
Comment (akintolga) [inline scripts/superdesk-publish/publish.js]: Would it better to use products service get all products?

Comment (marwoodandrew) [inline scripts/superdesk-publish/publish.js]: Probably

**Comment by akintolga** [comment]
:+1:

**Pull request merged**